newrelic-prometheus-configurator.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: newrelic-prometheus-configurator
+  version: 1.4.0
+  epoch: 0
+  description: New Relic Prometheus Configurator
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - automake
```

Comment:
`automake`  not needed here 
cc: @kaniini

Reply:
yes you are right, removed!